Clinical-Genomics/scout
# Refactors some buttons on variant page. Fix #1644

This PR fixes the problem that buttons are to short when dismissing etc.
It also refactors some html code for buttons

**How to test**:
1. Go to variant page and see that the dropdown does not reach all the way to the "save" button
<img width="441" alt="Skärmavbild 2020-02-10 kl  14 14 19" src="https://user-images.githubusercontent.com/405278/74152929-aaae6e00-4c0f-11ea-97db-75f152643ed1.png">

1. Install this branch
1. Check that the buttons reach all the way to save

**Expected outcome**:
Buttons reach all the way to save
Take a screenshot and attach or copy/paste the output.

**Review:**
